src/create_disk.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/sh
+
+export PATH=$PATH:/sbin:/usr/sbin
+
+usage() {
+	echo 'create_disk <device> <ostree-repo> <ostree-ref> <grub-script>'
+	exit 1
+}
+
+disk="$1"
+shift || usage
+ostree="$1"
+shift || usage
+ref="$1"
+shift || usage
+grub_script="$1"
+shift || usage
+
+set -e
+
+# partition and create fs
+sgdisk -Z $disk \
+	-n 1:0:+128M -c 1:boot \
+	-n 2:0:+128M -c 2:EFI-SYSTEM -t 2:C12A7328-F81F-11D2-BA4B-00A0C93EC93B \
+	-n 3:0:+128M -c 3:BIOS-BOOT  -t 3:21686148-6449-6E6F-744E-656564454649 \
+	-n 4:0:0     -c 4:root       -t 4:4F68BCE3-E8CD-4DB1-96E7-FBCAF984B709
```

Comment:
Random idea - change the order of the partitions on each build to avoid having people rely on it, and instead use the labels.

Reply:
I'm not sure how I feel about though I'm open to it. Regardless, lets punt that to a different PR/discussion?

Reply:
Marking this as resolved unless you really want it in this PR in particular.